underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystemFactory.java

```diff
@@ -68,34 +68,34 @@ public boolean supportsPath(String path) {
    *
    * @return true if simulation mode or if all required authentication credentials are present
    */
-  private boolean addAndCheckSwiftCredentials() {
+  private boolean addAndCheckSwiftCredentials(UnderFileSystemConfiguration conf) {
```

Comment:
I doublechecked git logs. I think there is a cleanup PR https://github.com/Alluxio/alluxio/pull/3740 long time ago which missed cleaning this swift UFS. Basically all the properties if they are already set in System Properties, they ought to be present in `Configuration` already, which is covered by `conf` here.
removed